examples/browser/package.json

```diff
@@ -21,11 +21,12 @@
     "build:backend": "tsc && npm run lint",
     "build:frontend": "webpack && cp src/frontend/index.html lib/frontend",
     "lint": "tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
-    "start": "concurrently --names watch-deps,backend,webpack-server --prefix \"[{name}]\" \"npm run watch:dependencies\" \"npm run start:backend\" \"npm run start:frontend\"",
+    "watch": "concurrently --names watch-deps,watch-backend,watch-frontend --prefix \"[{name}]\" \"npm run watch:dependencies\" \"npm run watch:backend\" \"npm run watch:frontend\"",
+    "start": "concurrently --names backend,frontend --prefix \"[{name}]\" \"npm run start:backend\" \"npm run start:frontend\"",
     "start:backend": "node ./lib/backend/main.js",
     "start:frontend": "webpack-dev-server --open",
     "cold:start": "npm run clean && npm run build && npm start",
-    "watch:backend": "tsc --watch",
+    "watch:backend": "tsc --watch -p ../../",
```

Comment:
Well it's another shell & another command it's running the file-updater anyway so like this all you would have to do is npm run watch and all would be taken care of ?